### PR TITLE
Decode Morse

### PR DESCRIPTION
**Pull request by ookino**
## Project Features

- Decodes a Morse code character, takes a string parameter, and return sthe corresponding character in uppercase (e.g. decode_char(".-") returns "A").
  -Decodes an entire word in Morse code, takes a string parameter, and return the string representation. Every character in a word will be separated by a single space (e.g. decode_word("-- -.--") returns "MY").
- Decodes the entire message in Morse code, takes a string parameter, and return the string representation. Every word will be separated by 3 spaces (e.g.

```ruby
decode("-- -.--   -. .- -- .")
```

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; returns **_"My Name"_**
and

```ruby
decode(".-   -... --- -..-   ..-. ..- .-.. .-..   --- ..-. / .-. ..- -... .. . ...")
```

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; returns the decoded message from the old bottle

### CODE REVIEW

File: decode.rb [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+def decode_char(char)
+  if MORSE.key? char
+    MORSE[char]
+  else
+    'Input is not morse code'
+  end
+end
+
+def dedcode_word(word)
```
Comment (rloterh) [inline decode.rb]: - [ ] Your app is throwing an error due to a mismatch in method definition and method call. Kindly ensure to improve upon this aspect

**Error:**
image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Okino & MB Taha,  

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦  🚦  

## Good Points 👍  
- Descriptive Pull Request
- Good ReadMe
- No linter errors

## Required Changes ♻️
_Kindly check the comments under the review._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: decode.rb [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  end
+  word_arr.join
+end
+
+def decode_message(msg)
+  morse_msg = ''
+  msg.strip.tr('/', ' ').split('   ').each do |n|
+    morse_msg += "#{decode_word(n)} "
+  end
+  word
```
Comment (petudeveloper) [inline decode.rb]: When trying to run your application I am having the following error: `undefined local variable or method `word'`
Please make the necessary changes to your code in order to correct this error. 

Check the image below for a better understanding:
Screenshot from 2022-01-20 20-08-29


**hint:** You are returning the wrong variable name `word`, the only variable you have inside your `decode_message` method is `hint: You are using`.

**Review by petudeveloper — CHANGES_REQUESTED** [review]
Hi @yaseerokino, 

Great work on making the changes requested by a previous reviewer 👏 

You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @yaseerokino and @mbdesigns1989,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**